### PR TITLE
#5648 Download statistics

### PR DESCRIPTION
**Pull request by JL102**
connects #5648

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,13 @@
 - To start the electron app: `npm run electron`
 
 ## Package
-- To build a package:`npm install && npm run ng-high-memory` and then: `npm run dist`
+- To build a package:`npm install --save-dev && npm run ng-high-memory` and then: `npm run dist`
 - The packages will be placed in `../output`
-- Example: `npm install && npm run build-prod && npm run dist` will make a production-build installer for Windows
+- Example: `npm install --save-dev && npm run build-prod && npm run dist` will make a production-build installer for Windows
```
Comment (rmroot) [inline README.md:20]: We don't want `--save-dev` in these instructions. That's for installing new packages to a project and saving them to the dependencies in package.json

File: package.json [hunk 2]
```diff
@@ -118,14 +119,15 @@
     "@types/lodash": "4.14.100",
     "@types/node": "^14.18.12",
     "angular-plotly.js": "^4.0.4",
+    "axios": "^0.27.2",
     "codelyzer": "^6.0.0",
     "electron": "13.6.6",
     "electron-builder": "22.9.1",
     "electron-download-tf": "4.3.4",
     "electron-packager": "15.2.0",
-    "jasmine-core": "~3.6.0",
```
Comment (rmroot) [inline package.json:126]: Why are we removing `jasmine-core` here?

**Review by rmroot — CHANGES_REQUESTED** [review]

File: scripts/download-stats.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { Parser } from 'json2csv';
```
Comment (nbintertech) [inline scripts/download-stats.ts:2]: See if papaparse (already a dependency) works for these needs and remove json2csv.

**Review by nbintertech — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (JL102) [reply package.json:126]: The specific version caused installation problems and it's not being directly used. It's already a sub-dependency, so it'll be installed anyways.

**Review by nbintertech — APPROVED** [review]

**Review by rmroot — APPROVED** [review]

**Pull request merged**